src/components/scene/reflection.js

```diff
@@ -45,7 +45,7 @@ module.exports.Component = register('reflection', {
       var renderer = self.el.renderer;
       var session = renderer.xr.getSession();
       if (
-        session.requestLightProbe && self.el.is('ar-mode')
+        session && session.requestLightProbe && self.el.is('ar-mode')
```

Comment:
can do an early return `if (!self.el.is('ar-mode')) { return; }`

Reply:
I have reworked the commit according to your suggestion :+1: